api/PayrollProcessor.Core.Domain/Intrastructure/Serialization/DefaultJsonSerializerSettings.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 
-namespace PayrollProcessor.Functions.Infrastructure
+namespace PayrollProcessor.Core.Domain.Infrastructure.Serialization
```

Comment:
I put this in the Domain to make sure we have consistent serialization configuration throughout the backend.